Sample/API/API.swift

```diff
@@ -73,11 +73,13 @@ public final class MovieAPI: APIProtocol {
     
     // weaver: urlSession <- URLSession
     
-    public init(injecting dependencies: MovieAPIDependencyResolver) {
+    // weaver: logger = Logger
+    
+    init(injecting dependencies: MovieAPIDependencyResolver) {
```

Comment:
And when I want to have a public init that takes in different properties?
For example the framework authorisation credentials 👉  `API.init(token: "unique-framework")`

Reply:
There's an annotation for parameters:
```
// weaver: token <= String
```

Reply:
Not an elegant solution because it does not force the user of the framework to init with the parameter. 

I have not tried the annotation out but I assume the usage would not result in the example I described above but something like:

`API.token = "unique-framework"`

Correct me if I'm wrong on this one. I want to have an option to designate a public init that by design eliminates the option to mess up and forget about aforementioned `token`

Reply:
It does force the user of the framework to init with the parameter.

I added an example to the sample in order to demonstrate that:

This line -> https://github.com/scribd/Weaver/blob/31-access-level/Sample/API/Manager/MovieManager.swift#L39

Generates this initializer -> https://github.com/scribd/Weaver/blob/31-access-level/Sample/API/Generated/Weaver.MovieManager.swift#L76-L78